### PR TITLE
[aform, atable] added file attach component

### PR DESCRIPTION
**Pull request by bhattdevarsh**
https://github.com/agritheory/stonecrop/issues/64

[aform, atable]

**AForm:**

https://github.com/agritheory/stonecrop/assets/58166671/1a759e7c-45a5-496c-a515-1bc4b74d9d73

**ATable:**

https://github.com/agritheory/stonecrop/assets/58166671/7be1ce2a-bcc9-472b-8270-4c1e2148d254

### CODE REVIEW

File: aform/src/components/form/AFileAttach.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { SchemaTypes } from 'types/index';
+
+const props = defineProps<{
+  schema: SchemaTypes[]
+  label: string
+  collapsible?: boolean
+  data?: any
+}>()
+
+const { files, open, reset } = useFileDialog()
```
Comment (agritheory) [inline aform/src/components/form/AFileAttach.vue]: @bhattdevarsh Question about using this in a client/server context: how do we call an upload API? Should we provide

Comment (bhattdevarsh) [reply aform/src/components/form/AFileAttach.vue]: May be we want to use `onChange` for that, as per the documentation that is only available in `10.9.0` and we are using `9.13.0`.

Ref: [10.9.0 usage](https://github.com/vueuse/vueuse/blob/main/packages/core/useFileDialog/index.ts#L45)
[9.13.0 Type Declaration](https://v9-13-0.vueuse.org/core/useFileDialog/#type-declarations)
[stonecrop package.json](https://github.com/agritheory/stonecrop/blob/development/stonecrop/package.json#L40)

Comment (agritheory) [reply aform/src/components/form/AFileAttach.vue]: Got it. This is related to resolving #120 in that case

Comment (Alchez) [reply aform/src/components/form/AFileAttach.vue]: @bhattdevarsh this one can move ahead with v10.9.0

**Comment by agritheory** [comment]
@bhattdevarsh Can you use the newly created `file-attach` branch on this repo? I rebased it so @crabinak can access it more easily.

**Comment by Alchez** [comment]
@agritheory @bhattdevarsh I'm closing this in favour of #138

**Pull request closed without merging**